src/Http/Requests/Organization/StoreOrganizationRequest.php

```diff
@@ -16,8 +16,8 @@ public function rules(): array
             "location" => ["required"],
             "organization_type" => ["required"],
             "foundation_date" => ["required"],
-            "number_of_employers" => ["required"],
-            "logo" => ["image", "max:2048"],
+            "number_of_employees" => ["required"],
+            "logo" => ["required", "image", "max:2048"],
```

Comment:
```suggestion
            "logo" => ["image", "max:2048"],
```
Logo is not required when we have default logo system.